pkg/visitors/cat.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package visitors
+
+import (
+	"regexp"
+
+	"github.com/linuxboot/fiano/pkg/uefi"
+)
+
+// Cat concatenates all RAW data sections from a file into a single byte slice.
+type Cat struct {
+	// Input
+	Predicate func(f *uefi.File, name string) bool
+
+	// Output
+	Buf     []byte
```

Comment:
Everything looks good, except this buffer never gets printed. Here's one way to fix it:

- Change this `Buf` to an `io.Writer`
- In the test initialize it to a `bytes.Buffer`
- In the RegisterCLI, initialize it to `os.stdout`

Reply:
My intent here is that some program will call this and manage the Buf by itself. I guess that does not with with how the commands work ... ok, I'll change it.

Reply:
done